navigator_engine/common/progress_tracker.py

```diff
@@ -33,6 +35,7 @@ def report_progress(self) -> dict:
             })
         self.report = {
             'progress': self.percentage_progress(),
+            'currentMilestoneID': current_milestone,
```

Comment:
That would look more natural, what do you think? `currentMilestoneID > currentMilestoneId`